src/ontology/metadata/doid.metadata.sssom.yml

```diff
@@ -6,32 +6,31 @@ license: https://creativecommons.org/licenses/by/4.0/
 creator_id: https://orcid.org/0000-0002-2906-7319
 curie_map:
   ICD10: http://apps.who.int/classifications/icd10/browse/2010/en#/
-  MedDRA: https://identifiers.org/meddra/
+  # MedDRA: https://identifiers.org/meddra/
   MESH: https://meshb.nlm.nih.gov/record/ui?ui=
   OMIM: https://omim.org/entry/
   OMIMPS: https://www.omim.org/phenotypicSeries/PS
-  Orphanet: http://www.orpha.net/ORDO/Orphanet_
+  # Orphanet: http://www.orpha.net/ORDO/Orphanet_
   UMLS: http://linkedlifedata.com/resource/umls/id/
   DOID: http://purl.obolibrary.org/obo/DOID_
   EFO: http://www.ebi.ac.uk/efo/EFO_
   ENVO: http://purl.obolibrary.org/obo/ENVO_
-  GARD: "https://bioregistry.io/reference/gard:"
+  GARD: "https://bioregistry.io/gard:"
   ICD10CM: https://icd.codes/icd10cm/
   ICD9CM: https://icd.codes/icd9cm/
-  ICD9CM_2005: https://icd.codes/icd9cm/
-  ICDO: "https://bioregistry.io/reference/gard:"
+  # ICD9CM_2005: https://icd.codes/icd9cm/
+  ICDO: "https://bioregistry.io/icdo:"
   KEGG: http://www.kegg.jp/entry/
   MEDDRA: https://identifiers.org/meddra/
   NCI: http://purl.obolibrary.org/obo/NCIT_
-  OMIT: http://purl.obolibrary.org/obo/ENVO_
+  OMIT: http://purl.obolibrary.org/obo/OMIT_
   ORDO: http://www.orpha.net/ORDO/Orphanet_
   SNOMEDCT_US_2020_03_01: http://identifiers.org/snomedct/
-  SNOMEDCT_US_2020_09_01: http://identifiers.org/snomedct/
-  SNOMEDCT_US_2021_07_31: http://identifiers.org/snomedct/
-  SNOMEDCT_US_2021_09_01: http://identifiers.org/snomedct/
-  SNOMEDCT_US_2022_07_31: http://identifiers.org/snomedct/
-  SNOMEDCT_US_2022_10_31: http://identifiers.org/snomedct/
-  SNOMEDCT_US_2023_02_28: http://identifiers.org/snomedct/
+  # SNOMEDCT_US_2020_09_01: http://identifiers.org/snomedct/
+  # SNOMEDCT_US_2021_07_31: http://identifiers.org/snomedct/
+  # SNOMEDCT_US_2021_09_01: http://identifiers.org/snomedct/
+  # SNOMEDCT_US_2022_07_31: http://identifiers.org/snomedct/
+  # SNOMEDCT_US_2022_10_31: http://identifiers.org/snomedct/
+  # SNOMEDCT_US_2023_02_28: http://identifiers.org/snomedct/
```

Comment:
@matentzn why are these commented out vs. removed or what are these comments a reminder of in the future?